src/components/patientForm/patientForm.tsx

```diff
@@ -27,18 +32,19 @@ export const PatientForm: React.FC = () => {
       },
     };
     dispatch(addPatientDetails(formatedData));
-    form.resetFields();
+    localStorage.setItem("patientData", JSON.stringify(patientData));
```

Comment:
Is there need to populate localStorage with patientData? Aren't we using redux to store patientData?

Reply:
I stored the data in localStorage to read the data as initialState.